src/main/java/com/zerobase/timate/service/CalendarMemberService.java

```diff
@@ -136,5 +147,58 @@ private void changeMaster(Long userId, Long newMasterId, Long calendarId) {
 	}
 
 
+	/**
+	 * 초대 코드 생성
+	 */
+	public String generateInviteLink(Long userId, Long calendarId) {
+		// 해당 사용자가 캘린더의 멤버인지 확인
+		UserCalendar userCalendar = commonService.getUserCalendar(userId, calendarId);
+		if (userCalendar.getCalendar().getType().equals(CalendarType.PRIVATE)) {
+			throw new CalendarException(CANNOT_INVITE_TO_PERSONAL_CALENDAR);
+		}
+
+		String calendarUuid = UUID.randomUUID().toString();
+        String redisKey = getInvitationRedisKey(calendarUuid);
+
+        // Redis에 초대 코드 저장 (유효기간 24시간)
+        redisTemplate.opsForValue().set(redisKey, calendarId.toString(), Duration.ofHours(24));
+
+        return "http://" + serverUrl + "/calendars/" + calendarId + "/members/invitation?code=" + calendarUuid;
+	}
+
+	/**
+	 * 초대 링크 검증
+	 */
+	public Long validateInvitation(String inviteCode) {
+		String redisKey = getInvitationRedisKey(inviteCode);
+		String calendarId = redisTemplate.opsForValue().get(redisKey);
```

Comment:
```suggestion
		String calendarId = redisTemplate.opsForValue().get(getInvitationRedisKey(inviteCode));
```
redisKey가 여려번 코드에서 사용되는 것이 아니라 한번만 사용된다면 별도의 변수로 선언하지 않는 것이 가독성이나 리소스측면에서 좋습니다